site/public/index.php

```diff
@@ -126,11 +126,12 @@ function error_handler() {
 }
 else {
     $response = WebRouter::getWebResponse($request, $core);
-}
+    $request_json = isset($_SERVER['HTTP_ACCEPT']) && $_SERVER['HTTP_ACCEPT'] === "text/json";
```

Comment:
`application/json` is the official mime-type for JSON per [RFC 4627](https://www.ietf.org/rfc/rfc4627.txt).